File: desktop/src/com/infeez/simple/desktop/DesktopLauncher.java
package com.infeez.simple.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.infeez.simple.Main;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.width = 400;
		config.height = 400;
		config.resizable = false;
		config.title = "Checkers !";
		new LwjglApplication(new Main(), config);
	}
}

File: desktop/src/main/kotlin/com/infeez/simple/desktop/DesktopLauncher.kt
package com.infeez.simple.desktop

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration
import com.infeez.simple.Main
import com.infeez.simple.utils.BoardConfig

fun main() {
    val config = Lwjgl3ApplicationConfiguration().apply {
        setTitle("Checkers")
        setWindowedMode(BoardConfig.BOARD_PIXEL_SIZE, BoardConfig.BOARD_PIXEL_SIZE)
        setResizable(false)
    }

    Lwjgl3Application(Main(), config)
}
